build/mac/tweak_info_plist.py

```diff
@@ -98,6 +98,9 @@ def Main(argv):
   # Explicitly disable profiling
   plist['SUEnableSystemProfiling'] = False
 
+  # Explicitly force user notification style to use alerts
+  plist['NSUserNotificationAlertStyle'] = "alert"
```

Comment:
There are other implications for this with web notifications. I think this needs more discussion/review as it changes behavior for the entire app

Reply:
@tmancey have you tested web notifications with this change? https://www.bennish.net/web-notifications.html
Also what happens if the user then changes the setting to banner? Do any notifications appear?

Reply:
Tested with web notifications which also do not dismiss until manually closed. With regards to changing the setting from Alerts to Banner in macOS/Preferences/Notifications then web notifications work as they did before the change

Reply:
If the user changes the setting to banner, they'll get the less reliable behavior.  Or the could turn it off.  Either is fine.  We just want it to work best in the default case.

Reply:
Requested change was reverted